src/components/ProfileBanner/Organization/index.js

```diff
@@ -105,10 +106,17 @@ export default function Banner({
               <Grid
                 item
                 xs={12}
-                md={3}
+                md={6}
                 container
                 className={classes.buttonContainer}
               >
+                <Link
```

Comment:
also add a check.
**Only and only if the organisation belongs to the user, then show this button**

Reply:
@ayush3160 please have a look at this